app/models/requests/request.rb

```diff
@@ -232,7 +232,7 @@ def load_items
       elsif visuals?
         return nil
       else
-        if @mfhd
+        if @mfhd && serial?
```

Comment:
@kevinreiss It looks like this added conditional has caused a number of tests to fail. I will update the tests on Monday so we can get this wrapped up.